kafka-client/src/main/java/dev/responsive/kafka/internal/stores/RemoteWindowOperations.java

```diff
@@ -106,6 +109,22 @@ public static RemoteWindowOperations create(
       case MONGO_DB:
         table = createMongo(params, sessionClients, partitioner, responsiveConfig);
         break;
+      case RS3:
+        final var responsiveMetrics = loadMetrics(appConfigs);
+        final var scopeBuilder = responsiveMetrics.storeLevelMetricScopeBuilder(
+            Utils.extractThreadIdFromThreadName(Thread.currentThread().getName()),
+            changelog,
+            params.name().tableName()
```

Comment:
I know we use `#tableName` for the existing caller of this method but I think it should actually be `#kafkaName` (here and in PartitionedOperations)

 We seem to use the kafka store name rather than the table name for these metrics elsewhere